tensorflow_addons/image/distort_image_ops_test.py

```diff
@@ -101,15 +100,18 @@ def test_adjust_random_hue_in_yiq(self):
                 y_tf = self._adjust_hue_in_yiq_tf(x_np, delta_h)
                 self.assertAllClose(y_tf, y_np, rtol=2e-4, atol=1e-4)
 
-    def test_invalid_shapes(self):
+    def test_invalid_rank(self):
+        msg = "Shape must be at least rank 3 but is rank 2"
         x_np = np.random.rand(2, 3) * 255.
         delta_h = np.random.rand() * 2.0 - 1.0
-        with self.assertRaises((tf.errors.InvalidArgumentError, ValueError)):
+        with self.assertRaisesRegex(ValueError, msg):
             self.evaluate(self._adjust_hue_in_yiq_tf(x_np, delta_h))
+
+    def test_invalid_channels(self):
+        msg = "Dimension must be 3 but is 4"
         x_np = np.random.rand(4, 2, 4) * 255.
         delta_h = np.random.rand() * 2.0 - 1.0
-        with self.assertRaisesOpError("input must have 3 channels "
-                                      "but instead has 4 channels"):
+        with self.assertRaisesRegex(ValueError, msg):
```

Comment:
I did not figure out why the check failure [here](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/custom_ops/image/cc/kernels/adjust_hsv_in_yiq_op.cc#L72) is not raised in the graph mode in the GPU version of tensorflow-2.0-preview, but it is raised in the CPU version.

Reply:
I also have no idea about it... I'd suppose it should either fail or pass regardless of the device.